src/medali/lib/harmonic_params/V1M1.ini

```diff
@@ -0,0 +1,98 @@
+# Harmonic parameters metadata, version 1.0
+
+[Metadata]
+
+# creation date - date of the data processing
+date_creation: datetime
+
+# modification date - date of last modification, by default set the same as the creation time when processing the data
+date_modification: datetime
+
+# dataset creator
+creator: string
+
+# name of the used worker software
+worker_name: string
+
+# tag of the used worker software
+worker_git_tag: string
+
+# 7-digit commit of the used worker software
+worker_git_commit: string
+
+# name of the used wrapper software
+wrapper_name: string
+
+# tag of the used wrapper software
+wrapper_git_tag: string
+
+# 7-digit commit of the used wrapper software
+wrapper_git_commit: string
+
+# input data version version of the input data used for harmonic regression computation (e.g. V01R01)
+input_data_version: string
```

Comment:
Is the new version not accepting multiple ones? Then it should be plural and a list, you can check the latest version in` tempinator`.